readme.md

```diff
@@ -0,0 +1,59 @@
+# **Bijutsu Ecommerce**
+
+## What is Bijutsu ?
+
+Bijutsu is a japanese word for art. And thats what we sell at bijutsu from digital art, music, photos, gifs and all the things related to art.
+
+**To visit us go to : [Bijutsu Ecommerce](https://bijutsu.netlify.app/index.html 'Bijutsu Landing Page')**
+
+# Pages
+
+- [SignUp](https://bijutsu.netlify.app/pages/authentication/signup.html 'SignUp')
+- [Login](https://bijutsu.netlify.app/pages/authentication/login.html 'LogIn')
+- [Landing](https://bijutsu.netlify.app/index.html 'Bijutsu Landing Page')
+- [Product List](https://bijutsu.netlify.app/pages/product_listing/product_listing.html 'Product List')
+- [Wishlist](https://bijutsu.netlify.app/pages/wishlist/wishlist.html 'Wishlist')
+- [Cart](https://bijutsu.netlify.app/pages/cart/cart.html, 'Cart')
```

Comment:
link not working, remove the ',' from the link

Reply:
done

Reply:
and footer has a lot of empty space because height is 40vh. Otherwise very good design.